nameko/concurrency/__init__.py

```diff
@@ -96,6 +99,21 @@
     )
 
 
+def monkey_patch_if_enforced():
+    config_value = config.get(
+        MONKEY_PATCH_ENFORCED_CONFIG_KEY,
+        DEFAULT_MONKEY_PATCH_ENFORCED
+    )
+    config_value = str(config_value).lower()
+    monkey_patch_enforced = {
+        'false': False,
+        'true': True,
+    }[config_value]
```

Comment:
Doesn't YAML take care of this boolean handling for you?

Reply:
yes, this was initially designed to handle environment variables ... but I guess now that it's just a config value this isn't necessary.  Changed it